src/core/device.c

```diff
@@ -766,6 +766,26 @@ static int device_dispatch_io(sd_event_source *source, int fd, uint32_t revents,
                 return 0;
         }
 
+        if (streq(action, "change"))  {
+                _cleanup_free_ char *e = NULL;
+                Unit *u;
+
+                r = unit_name_from_path(sysfs, ".device", &e);
+                if (r < 0)
+                        return log_error_errno(r, "Failed to generate unit name from device path: %m");
+
+                u = manager_get_unit(m, e);
+                if (u) {
+                        if (UNIT_VTABLE(u)->active_state(u) == UNIT_ACTIVE) {
+                                r = manager_add_job(m, JOB_RELOAD, u, JOB_REPLACE, NULL, NULL);
+                                if (r < 0) {
+                                        log_error_errno(r, "Failed to add job to manager : %m");
```

Comment:
The space before ':' should go.

Reply:
bad habit of mine, will fix

This is a difference between french and english typo convention. In french you always have a space before and after "tall" punctuation signs.